rllib/utils/test_utils.py

```diff
@@ -431,40 +450,82 @@ def _test(
                     "for that!"
                 )
 
-    # Loop through: Policy vs Algorithm; Different API methods to calculate
-    # actions; unsquash option; clip option; full fetch or not.
-    for what in [pol, algorithm]:
+    def _test_compute_actions_from_raw_data(
```

Comment:
This method only checks if we are able to compute actions without an error and if the shape fits actions space.
Actions are not numerically compared against one another.